.vscode/settings.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
please add .vscode to your `.gitignore` file.
Those files has nothing to do with the PR

Reply:
As you can see - this file is still included in branch.